.gitignore

```diff
@@ -14,7 +14,10 @@ _test
 /gitea_*
 
 # MS VSCode
-.vscode
+.vscode/*
+!.vscode/settings.json
+!.vscode/launch.json
+!.vscode/extensions.json
```

Comment:
```suggestion
.vscode
```

The gitignore could be kept as-is. It doesn't affected the manually added files.

Reply:
yes but this is the more compliant way!

Reply:
I do not think it is more compliant. Instead it just makes thing more complicated.

By using a single `.vscode`, everything just works well, and the manually added files are always correctly handled.